test/vtgatev3_test.py

```diff
@@ -888,6 +888,20 @@ def test_user2(self):
         {'id': 7})
     vtgate_conn.commit()
 
+    # Test multi shard delete
+    vtgate_conn.begin()
+    result = self.execute_on_master(
+        vtgate_conn,
+        'insert into vt_user (id, name) values (:id0, :name0),(:id1, :name1)',
+        {'id0': 22, 'name0': 'name2', 'id1': 33, 'name1': 'name2'})
+    self.assertEqual(result, ([], 2L, 0L, []))
+    result = self.execute_on_master(
```

Comment:
Perhaps this is overkill but I think it would make sense to add a test that the rows exist after insert but are actually deleted by the delete :)

Reply:
The `2L` asserts that two rows were affected by the delete, so I think adding the select will be redundant. What do you think ?

Reply:
Ah ok. Yeah that seems legit enough.

Reply:
75